core/src/main/java/de/jplag/JPlag.java

```diff
@@ -96,4 +101,27 @@ private static void logSkippedSubmissions(SubmissionSet submissionSet, JPlagOpti
             }
         }
     }
+
+    private static void checkForConfigurationConsistency(JPlagOptions options) throws ConfigurationException {
+        List<String> duplicateNames = getDuplicateSubmissionFolderNames(options);
+        if (duplicateNames.size() > 0) {
+            throw new ConfigurationException(String.format("Duplicate root directory names found: %s", String.join(", ", duplicateNames)));
```

Comment:
We already have a `RootDirectoryException`, that would fit here as well.

Reply:
My plan was too keep it in line with #1479.
But if you prefer that exception I can also implement it that way

Reply:
For that PR, I want to move away from using an exception and just log and error, so thus `RootDirectoryException` fits better in your PR.